services/worker/src/worker/job_manager.py

```diff
@@ -107,6 +107,7 @@ def run_job(self) -> JobResult:
                 "job_runner_version": self.job_runner_version,
                 "is_success": False,
                 "output": None,
+                "duration": get_duration_or_none(self.job_info["started_at"]),
```

Comment:
this is dumb but otherwise mypy warns that `Optional[datetime]` is being passed because job_info["started_at"] might be None if a job is not started yet.